sigar_windows.go

```diff
@@ -542,11 +542,103 @@ func (self *NetUdpV6ConnList) Get() error {
 }
 
 func (self *NetProtoV4Stats) Get() error {
-	return notImplemented()
+	protoV4Queries := []string{
+		`\TCPv4\Connections Active`,
+		`\TCPv4\Connections Passive`,
+		`\TCPv4\Connection Failures`,
+		`\TCPv4\Connections Reset`,
+		`\TCPv4\Connections Established`,
+
+		`\UDPv4\Datagrams Received Errors`,
+
+		`\IPv4\Datagrams Received Header Errors`,
+		`\IPv4\Datagrams Received Address Errors`,
+		`\IPv4\Datagrams Received Discarded`,
+		`\IPv4\Datagrams Received Unknown Protocol`,
+		`\IPv4\Datagrams Outbound Discarded`,
+		`\IPv4\Datagrams Outbound No Route`,
+
+		`\ICMP\Messages Received Errors`,
+		`\ICMP\Received Dest. Unreachable`,
+		`\ICMP\Messages Outbound Errors`,
+		`\ICMP\Sent Destination Unreachable`,
+	}
+
+	results, err := runRawPdhQueries(protoV4Queries)
+	if err != nil {
+		return err
+	}
+	self.TCP.ActiveOpens, results = uint64(results[0]), results[1:]
```

Comment:
Currently, `runRawPdhQueries` fails if any counter value can't be obtained, so I don't think there is a risk of parsing an undersized slice that would cause a panic. But your point is well-taken - ROCANA-10125 should address this.